src/styles/blocks/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  justify-content: space-between;
+  background-color: $white;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    font-size: 0;
+  }
+}
+
+.nav {
+  margin: 0;
+  
+  &__list {
+    display: flex;
+    text-transform: uppercase;
+    margin: 0;
+    padding: 0;
```

Comment:
reset all margins and paddings in correspondent tag selectors (e. g. p, h1, h2, ... {margin: 0; padding: 0;}